.releaserc

```diff
@@ -1,8 +1,22 @@
 {
   "tagFormat": "${version}",
+  "branches": [
+    { "name": "master", "prerelease": false },
+    { "name": "development", "prerelease": "beta" },
+    { "name": "feature/*", "prerelease": "alpha" }
+  ],
```

Comment:
from my understanding from https://semantic-release.gitbook.io/semantic-release/recipes/release-workflow/pre-releases
this is the right feature for us. it also means we can remove the release logic in the github action to change the stage

what I would understand you want to achieve what we want is
```
  "branches": [
    { "name": "master", "prerelease": false },
    { "name": "development", "prerelease": "true" },
  ],
  ```